### ISSUE TITLE
it's not "native", it's "vanilla"

### ISSUE THREAD

**Issue by noformnocontent** [z-pattern-matching/z#26]
"native" nowadays means "a system app" -- mobile or desktop, i.e. "does not depend on a web-browser"

what you wanted to say is "vanilla" -- b/c it dosnt need babel, webpack or a WASM compiler

**Comment by leonardiwagner** [comment]
Thanks, I'm gonna change it.

**Comment by ForsakenHarmony** [comment]
native for me would also include if it was a language feature, but it's not, more of a hack

**Comment by depoulo** [comment]
In this spirit, are you specifically trying to align as closely as possible to https://github.com/tc39/proposal-pattern-matching?

**Comment by leonardiwagner** [comment]
@depoulo No. That proposal is not mature yet, there are a lot of caveats, besides not being able to be close anyway due the proposal features rely mostly on Javascript syntax changes.

**Comment by naomik** [comment]
Still says "native" on the front-page https://z-pattern-matching.github.io/

It's not native. It's vanilla.

**Comment by ForsakenHarmony** [comment]
It still says native

**Comment by leonardiwagner** [comment]
I don't know why this issue was closed, reopening right now, thanks for the warning @ForsakenHarmony and @naomik 

Just to remember this project isn't "mine" and it's open to anyone send PRs, I'm currently assigned to update the whole website (including new features done past weeks) which is a higher priority, but after that, I can work on this also if still this description. Thanks.

**Comment by ForsakenHarmony** [comment]
it's on the website
